### DISCUSSION TITLE
Syrian flag

### DISCUSSION THREAD

**Discussion by Eng-Ahmet** [jackocnr/intl-tel-input#1904] [General]
Hello, I would like to ask you to change the Syrian flag logo only. It has become like this.

![flag-sirii-1932-1958-1961-1963-1024x512](https://github.com/user-attachments/assets/d1c4d579-c1c3-4bb4-93b3-ef6f3651abe8)

**Comment by jackocnr** [comment]
Hi there, thanks for the suggestion. We use [flag-icons](https://github.com/lipis/flag-icons) for our flags, so you would need to open an issue with them to update their flag first. I see there are already pending pull requests to do this, so maybe add your voice to one of those!

**Comment by jackocnr** [comment]
I've just released v25.3.0 with this change included 👍🏻
